spec/root-apps/webpack2-test.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
This webpack config creates the bundle used in the karma.conf to test all the webpack things.